aspnetcore/tutorials/publish-to-azure-webapp-using-cli.md

```diff
@@ -0,0 +1,149 @@
+---
+
+# Deploy an ASP.NET Core application to Azure App Service from the command line
+
+By [Cam Soper](https://twitter.com/camsoper)
+
+This tutorial will show you how to build and deploy an ASP.NET Core application to Microsoft Azure App Service using command line tools.  When finished, you'll have a web application built in ASP.NET MVC Core hosted as an Azure App Service Web App.  This tutorial is written using Windows command line tools, but can be applied to macOS and Linux environments, as well.  
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+> * Create an Azure App Service website using Azure CLI
+> * Deploy an ASP.NET Core application to Azure App Service using the Git command line tool
+
+## Prerequisites
+
+To complete this tutorial, you'll need:
+
+* A [Microsoft Azure subscription](https://azure.microsoft.com/free/)
+* [.NET Core](https://www.microsoft.com/net/download/core)
+* [Git](https://www.git-scm.com/) command line client
+
+## Create a web application
+
+Create a new directory for the web application, create a new ASP.NET Core MVC application, and then run the website locally.
+
+```cmd
+REM Create the directory and navigate to it.
+REM These commands are Windows-specific. The Bash equivalents are mkdir and cd.
+md MyApplication
+cd MyApplication
+
+REM Create a new ASP.NET Core MVC application
+dotnet new mvc
+
+REM Run the application
+dotnet run
+```
+
+![Command line output](publish-to-azure-webapp-using-cli/_static/new_prj.png)
+
+You can then test the application by browsing to the displayed address (usually `http://localhost:5000`).
+
+![The website running locally](publish-to-azure-webapp-using-cli/_static/app_test.png)
+
+
+## Create the Azure App Service instance
+
+Using the [Azure Cloud Shell](/azure/cloud-shell/quickstart), create a resource group, App Service plan, and an App Service web app.
+
+```azurecli-interactive
+# Generate a unique Web App name
+let randomNum=$RANDOM*$RANDOM
+webappname=tutorialApp$randomNum
+
+# Create the DotNetAzureTutorial resource group
+az group create --name DotNetAzureTutorial --location EastUS
+
+# Create an App Service plan.
+az appservice plan create --name $webappname --resource-group DotNetAzureTutorial --sku FREE
+
+# Create the Web App
+az webapp create --name $webappname --resource-group DotNetAzureTutorial --plan $webappname
+```
+
+Before you deploy, you need to set the account-level deployment credentials.  Use the script below, making sure to include your own values for the user name and password.
+
+```azurecli-interactive
+az webapp deployment user set --user-name <desired user name> --password <desired password>
+```
+
+You'll also need the deployment URL to deploy the application using Git.  Retrieve the URL like this.
+
+```azurecli-interactive
+az webapp deployment source config-local-git -n $webappname -g DotNetAzureTutorial --query [url] -o tsv
+```
+Note the displayed URL ending in `.git`.  You'll need it in the next step.
```

Comment:
Total ban on future tense.
You need it in the next step. But better yet, get rid of you.

Note the displayed URL ending in `.git`; it's used it in the next step.